src/main/resources/db/migration/table_initialization.sql

```diff
@@ -0,0 +1,69 @@
+create schema "advisory";
+set schema 'advisory';
+
+create type phone_numbers_type as enum ('home', 'work', 'mobile');
+create type advisor_role as enum ('associate', 'partner', 'senior');
+create type application_status as enum ('new', 'assigned', 'on_hold', 'approved', 'canceled', 'declined');
+create type user_type as enum ('advisor', 'applicant');
+
+
+create table if not exists users
+(
+    id        bigserial,
+    email     text not null,
+    user_name text not null,
+    constraint users_pk primary key (id),
+    constraint email_uq unique (email),
+    constraint user_name_uq unique (user_name)
+);
+
+create table if not exists advisors
+(
+    id   bigserial,
+    role advisory.advisor_role not null,
+    constraint advisors_pk primary key (id)
+);
+
+create table if not exists applicants
+(
+    id                     bigserial,
+    first_name             text    not null,
+    last_name              text    not null,
+    social_security_number integer not null
```

Comment:
CCN might have leading zeros and according to wikipedia it has size of 9 [source](https://en.wikipedia.org/wiki/Social_Security_number#:~:text=The%20Social%20Security%20number%20is,four%20digits%2C%20the%20serial%20number.). What do you think about changing `integer` to `varchar(9)` here?